src/axom/sidre/core/DataStore.cpp

```diff
@@ -618,6 +622,54 @@ bool DataStore::generateBlueprintIndex(const std::string& domain_path,
   return success;
 }
 
+#ifdef AXOM_USE_MPI
+bool DataStore::generateBlueprintIndex(MPI_Comm comm,
+                                       const std::string& domain_path,
+                                       const std::string& mesh_name,
+                                       const std::string& index_path)
+{
+  Group* domain;
+  if (domain_path == "/")
```

Comment:
Should we also handle the empty string (`domain_path == ""`) as the root?
(or perhaps that case is already handled by the `else if` case)

Reply:
I think it might be an undocumented feature that "/" represents the root in sidre?  I'm not sure what would happen if an empty string is passed right now.  I could make "" also work for the root for this method, but I'm not sure it will be consistent with everything else in sidre.  Either way, the documentation should be clarified.